all/build.gradle

```diff
@@ -24,12 +24,13 @@ def subprojects = [
     project(':grpc-xds'),
 ]
 
-for (subproject in rootProject.subprojects) {
+for (subproject in subprojects) {
     if (subproject == project) {
         continue
     }
     evaluationDependsOn(subproject.path)
 }
+evaluationDependsOn(':grpc-interop-testing')
```

Comment:
Seems should also add
```
dependsOn(project(':grpc-interop-testing').jacocoTestReport.dependsOn)
```

in `task jacocoMerge(type: JacocoMerge) {}`

although we haven't excluded generated code in `:grpc-interop-testing` for jacoco.

Reply:
I agree, although technically unrelated to this change. You fine if I do it as a follow-up?

Reply:
Created #8093